loops-and-arrays.js

```diff
@@ -4,8 +4,10 @@
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 namen zou bevatten!
 // ==========================================
 
-
-
+const names = ["Henk", "Piet", "Fred", "Joop" /*heb zelf extra namen toegevoegd om script te testen*/, "kip", "kip", "kip", "kip"];
+for (let i = 0; i < names.length; i++) {
+    console.log(names[i]+ "je");
```

Comment:
De gedachte is goed, maar hiermee pas je niet de array zelf aan. Je plakt er als het ware tijdelijk iets bij. Wanneer je bijvoorbeeld op regel 11 nu console.log(names) zou typen, krijg je nog steeds de namen te zien zonder "je". 
Probeer in je for-loop de array zélf kan aanpassen, zodat elke naam wordt aangepast.
Je kan als check voor je zelf eerst de array loggen, daarna de for-loop draaien, en daarna weer de array loggen. Wat er op regel 14 tm 16 staat!